acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/ManualInterlockSigningAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+import tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.Eth2SigningRequestBody;
+import tech.pegasys.web3signer.core.signing.KeyType;
+import tech.pegasys.web3signer.dsl.utils.Eth2RequestUtils;
+import tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.response.Response;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled("Requires access to Interlock on USB Armory II")
+public class ManualInterlockSigningAcceptanceTest extends SigningAcceptanceTestBase {
+  private static final MetadataFileHelpers metadataFileHelpers = new MetadataFileHelpers();
+
+  // following keys are expected at /bls/key1.txt...key11.txt in USB Armory
+  private final List<BLSSecretKey> blsSecretKeys =
+      List.of(
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "73d51abbd89cb8196f0efb6892f94d68fccc2c35f0b84609e5f12c55dd85aba8")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "39722cbbf8b91a4b9045c5e6175f1001eac32f7fcd5eccda5c6e62fc4e638508")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "4c9326bb9805fa8f85882c12eae724cef0c62e118427f5948aefa5c428c43c93")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "384a62688ee1d9a01c9d58e303f2b3c9bc1885e8131565386f75f7ae6ca8d147")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "4b6b5c682f2db7e510e0c00ed67ac896c21b847acadd8df29cf63a77470989d2")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "13086d684f4b1a1632178a8c5be08a2fb01287c4a78313c41373701eb8e66232")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "25296867ee96fa5b275af1b72f699efcb61586565d4c3c7e41f4b3e692471abd")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "10e1a313e573d96abe701d8848742cf88166dd2ded38ac22267a05d1d62baf71")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "0bdeebbad8f9b240192635c42f40f2d02ee524c5a3fe8cda53fb4897b08c66fe")),
+          BLSSecretKey.fromBytes(
+              Bytes32.fromHexString(
+                  "5e8d5667ce78982a07242739ab03dc63c91e830c80a5b6adca777e3f216a405d")));
+
+  @BeforeEach
+  void init() {
+    final Path knownServersFile = testDirectory.resolve("interlockKnownServer.txt");
+    for (int i = 1; i <= 11; i++) {
```

Comment:
what is the magic 11? Is that the number of keys we have? Can we remove the magic number and put in what it represents?